ckanext/gla/templates/snippets/package_list.html

```diff
@@ -14,21 +14,29 @@
     
     
     {% block package_list %}
+    {% asset "gla/gla-search_result_logging" %}
         {% if packages %}
             {% block package_list_inner %}
+            {% set is_search = h.is_search_results_page(request) %}
             {% if h.show_favourite_datasets(g.userobj, request) %}
                 {% set favourites = h.get_followed_datasets(g.userobj, request) %}
                 {% if favourites %}
                     {% snippet 'snippets/favourites.html', favourites=favourites %}
                 {% endif %}
             {% endif %}
             {% set nonfavourites = h.remove_favourites(g.userobj, request, packages) %}
-                <ul class="{{ list_class or 'dataset-list list-unstyled' }}">
+                <ul class="{{ list_class or 'dataset-list list-unstyled' }}" data-module="gla_search_result_logging">
```

Comment:
I'm not sure the `data-module` is doing anything here.`search_result_logging.js` doesn't start with: `ckan.module('gla_search_result_logging',...)` so I don't think a module with that name gets registered with CKAN.

I don't think the module is really necessary anyway if you're just calling `logResultClick` directly in `package_item`
```suggestion
                <ul class="{{ list_class or 'dataset-list list-unstyled' }}">
```